packages/teams-js/src/private/videoEx.ts

```diff
@@ -244,6 +249,20 @@ export namespace videoEx {
     }
   }
 
+  function createMonitoredVideoFrameHandler(
```

Comment:
the name of this function seems misleading if the monitor is optional. It seems like it could result in instances like what you have here, where you call a function called createMonitoredVideoFrameHandler but the handler you are making is actually unmonitored because no monitor was passed in.

Reply:
updated, make it required